OloEngine/src/OloEngine/Asset/MeshCache.cpp

```
@@ -0,0 +1,366 @@
+                            if (ec)
+                            {
+                                allRemoved = false;
+                                ec.clear();
+                            }
+                        }
+                    }
+                    if (ec)
+                    {
+                        allRemoved = false;
+                    }
+                    if (allRemoved)
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)